File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        if len(nums) == 2:
            return [0,1]
        
        p = 0
        
        while p < len(nums)-1:
             
            if target - nums[p] in nums[p+1:]:
                return [p,nums[p+1:].index(target - nums[p])+p+1]
                    
            p += 1
        
        return [p,len(nums)-1]
            
            
        

File: 1-two-sum/1-two-sum.c


/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target, int* returnSize){
    
    *returnSize = 2;
    //정답용 array 만들기
    int* answer = (int *)malloc(sizeof(int) * 2);
    
    for(int i = 0;i < numsSize-1;i++){
        for(int j = i+1; j < numsSize;j++){
            if(nums[i] + nums[j] == target){
                answer[0] = i;
                answer[1] = j;
                return answer;
            }
        }
    }
    return answer;
}
    